buildSrc/src/main/java/Dependencies.kt

```diff
@@ -96,6 +96,7 @@ object Deps {
     const val mozilla_feature_qr = "org.mozilla.components:feature-qr:${Versions.mozilla_android_components}"
     const val mozilla_feature_search = "org.mozilla.components:feature-search:${Versions.mozilla_android_components}"
     const val mozilla_feature_session = "org.mozilla.components:feature-session:${Versions.mozilla_android_components}"
+    const val mozilla_feature_sync = "org.mozilla.components:feature-syncedtabs:${Versions.mozilla_android_components}"
```

Comment:
```suggestion
    const val mozilla_feature_syncedtabs = "org.mozilla.components:feature-syncedtabs:${Versions.mozilla_android_components}"
```

This naming can be misleading because "sync" is used to reference the product Firefox Sync and Synced Tabs is one of features it provides.